core/storage/config/gae_models.py

```diff
@@ -180,3 +180,67 @@ def create(
             rules=rule_dicts,
             rule_schema_version=rule_schema_version,
             default_value=default_value)
+
+
+class FeatureFlagModel(base_models.BaseModel):
+    """A class that represents named dynamic feature-flag.
+    This model only stores fields that can be updated in run time.
+
+    The id is the name of the feature-flag.
+    """
+
+    force_enable_for_all_users = datastore_services.BooleanProperty(
```

Comment:
Have the first two fields be indexed=True so that we can query by them if needed. (No harm.)

Reply:
Done